src/utils/__tests__/formatNumber.test.ts

```diff
@@ -1,13 +1,149 @@
-describe('Appease Jest', () => {
-  it('adds at least one test to the test file', () => {
-    expect(true).toBe(true)
+import { formatAmount } from '@/utils/formatNumber'
```

Comment:
Great tests! We should add another that checks that the "Thousands and decimal separators are used according to user's locale."

Reply:
We've decided before to go with literal assertions. Please refer to https://github.com/safe-global/safe-react/pull/3920/commits/09a66b717f763b821719e34edcb7a1ceb33b7c9b#r893219929